cvat/settings/base.py

```diff
@@ -458,8 +457,6 @@ def add_ssh_keys():
 LOCAL_LOAD_MAX_FILES_SIZE = 512 * 1024 * 1024  # 512 MB
 
 RESTRICTIONS = {
```

Comment:
Should be removed the whole dict RESTRICTIONS?

Reply:
Still used here https://github.com/opencv/cvat/blob/develop/cvat/apps/iam/permissions.py#L307,
Will be better to rename to IAM_PERMISSION_SETTINGS or smth like that?